recirq/quantum_chess/quantum_board_test.py

```diff
@@ -592,9 +592,10 @@ def test_pawn_capture(board):
         (u.squares_to_bitboard(['e8', 'f8', 'h8']), 'e8', 'g8'),
         (u.squares_to_bitboard(['e8', 'g8', 'h8']), 'e8', 'g8'),
         (u.squares_to_bitboard(['e8', 'f8', 'g8', 'h8']), 'e8', 'g8'),
+        (u.squares_to_bitboard(['e1', 'b1', 'a1']), 'e1', 'c1'),
         (u.squares_to_bitboard(['e1', 'c1', 'a1']), 'e1', 'c1'),
         (u.squares_to_bitboard(['e1', 'd1', 'a1']), 'e1', 'c1'),
-        (u.squares_to_bitboard(['e1', 'c1', 'd1', 'a1']), 'e1', 'c1'),
+        (u.squares_to_bitboard(['e1', 'b1', 'c1', 'd1', 'a1']), 'e1', 'c1'),
```

Comment:
Why do we need the extra b1 in this case?

Reply:
I think for queen side castle we need to test the extra column "b", comparing with king side castle.